product_docs/docs/biganimal/release/using_cluster/03_modifying_your_cluster/05_db_configuration_parameters.mdx

```diff
@@ -6,15 +6,45 @@ The database parameters listed on the DB Configuration tab are also referred to
 
 The list of parameters is populated based on the type of database you selected on the **Operational Settings** tab when you created your cluster. For more information about the parameters for your database type:
 
--   For Postgres parameters, see [Setting Parameters](https://www.postgresql.org/docs/current/config-setting.html) and [Server Configuration](https://www.postgresql.org/docs/current/runtime-config.html) in the Postgres documentation.
+- For PostgreSQL parameters, see [Setting Parameters](https://www.postgresql.org/docs/current/config-setting.html) and [Server Configuration](https://www.postgresql.org/docs/current/runtime-config.html) in the Postgres documentation.
      
 
--   For EDB Postgres Advanced Server, see [Summary of Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/02_summary_of_configuration_parameters/#summary_of_configuration_parameters) and [Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/).
+- For EDB Postgres Advanced Server, see [Summary of Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/02_summary_of_configuration_parameters/#summary_of_configuration_parameters) and [Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/).
+
+-  For more information on the types of values parameters accept, see [Parameter Names and Values](https://www.postgresql.org/docs/current/config-setting.html#CONFIG-SETTING-NAMES-VALUES). The types apply to both PostgreSQL and EDB Postgres Advanced Server parameters.
+
 
 !!!note
 Not all database configuration parameters are supported by BigAnimal. Some parameters, such as `wal_level` and `restore_command`, are reserved for EDB to provide the managed database features of BigAnimal. 
 !!!
 
+## Using formulas for parameter values
+
+In addition to entering specific values for parameters, for some parameters you can specify formulas to automatically calculate a value. You can use formulas for parameters of type integer and real using the following operators: `+ - / * > >= < <= == != && || ! ? : ( )`. GUCs used in formulas must also be of type integer or real. 
+
+BigAnimal has what we refer to as psuedo GUCs to facilitate creating equations. These read-only GUCs are:
+- `cpu_cores`
+- `disk_iops`
+- `disk_size`
+- `ram`
+- `replica_count` (node count)
+
+`ram` and `disk_size` have the default unit of KB. 
```

Comment:
It seems like this section would read better if it was written something like:

BigAnimal has what we refer to as psuedo GUCs to facilitate creating equations. These read-only GUCs are:
- `cpu_cores`: the number of cores provided by the VM running the cluster nodes
- `disk_iops`: the number of IOPS allocated to the disks backing the primary data volume
- `disk_size`: the amount of memory backing the primary data volume, in KB
- `ram`: the amount of memory provided by the VM running the cluster nodes, in KB
- `replica_count`: the total number of nodes in the cluster